docs/program-setup/build_rules.md

```diff
@@ -68,6 +69,22 @@ The feature labeled directories are used for software that implements functional
 
 In the above example, `mbed compile` includes files in directories named `FEATURE_BLE`, and not directories such as `FEATURE_STORAGE` or `FEATURE_CRYTOCELL310`.
 
+### Experimental directories
+
+By default, Mbed OS doesn't compile [experimental APIs](). You can override this behaviour by explicitly including "EXPERIMENTAL_API" in `mbed_app.json`:
+
+```
+{
+    “target_overrides” : {
+        “*” : {
+            “target.features_add” : [“EXPERIMENTAL_API”]
+     }
+  }
+}
+
+```
+<!--so for a specific one, or does this now allow me to use all experimental APIs in one giant go?-->
```

Comment:
I think it's worth having each set of experimental APIs having its own feature switch as well. So, for example, in order to use PSA, you would need to add both the EXPERIMENTAL_API and PSA features toggles.

Reply:
I think that's a decision for @bulislaw

Reply:
We discussed this ages ago, if I remember correctly, the consensus was behind "case by case". That is if the feature would have it's own flag it should be there hidden behind the master experimantal flag, so that when the API is stabilised no changes to the application are needed.